Gemfile.lock

```diff
@@ -0,0 +1,21 @@
+PATH
+  remote: .
+  specs:
+    schmooze (0.2.0)
+
+GEM
+  remote: https://rubygems.org/
+  specs:
+    minitest (5.22.3)
+    rake (13.2.1)
```

Comment:
> I can't run script/test or bundle exec rake due to an error.

Hmm, I couldn't run into this error. I bumped rake to a version that has the patch that was linked - does that for you?